src/components/action-button.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { Tooltip } from 'antd';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+export const ACTION_BUTTON_INTENT = {
+  default: 'default',
+  info: 'info',
+  success: 'success',
+  warning: 'warning',
+  error: 'error'
+};
+
+export function ActionButtonGroup({ children }) {
```

Comment:
Not sure what you think about this. Normally we do not export multiple components per file. But as this is just a simple container I thought we could leave them together with the "items" component.

Reply:
I also find it fine in this case.